db/seed.js

```diff
@@ -0,0 +1,28 @@
+const ORM = require('./index.js');
```

Comment:
an all caps variable name indicates "this is a constant; please don't change me." I would advise changing this var name